VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
``fill_parent`` is deprecated. Should be using ``match_parent`` 

http://stackoverflow.com/questions/5761960/what-is-the-difference-between-match-parent-and-fill-parent